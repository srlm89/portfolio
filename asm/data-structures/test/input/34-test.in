ERASE actual/34-test.out
INSERT 34 0.063821 Beauty And The Beast    720p
PRINT_IN_ORDER actual/34-test.out
INSERT 17 7.955046 Breach 1080p
INSERT 31 0.611211 Constantine HDDVD 1080p DTS x264
REMOVE_ID 31
PRINT_IN_ORDER actual/34-test.out
INSERT 2 9.286223 Apollo 13   1080p
INSERT 10 4.598667 Stardust   720p
INSERT 33 4.388398 Afro samurai uncut 720p
REMOVE_ID 33
PRINT_IN_ORDER actual/34-test.out
INSERT 6 1.363797 Click   720p
INSERT 38 7.518813 SAW V
INSERT 12 0.713867 Romeo And Juliet   720p
INSERT 24 6.745695 Eragon 720p
REMOVE_ID 12
REMOVE_ID 34
PRINT_IN_ORDER actual/34-test.out
INSERT 39 0.406099 Rocky Balboa    720p
INSERT 23 8.554752 Terminator 2 Judgement Day Directors Cut [1991]   1080p
INSERT 11 9.508342 Horseman    720p
INSERT 1 9.906953 X-Men 3   1080p
INSERT 42 2.095395 Species   720p
REMOVE_ID 23
REMOVE_ID 17
PRINT_IN_ORDER actual/34-test.out
INSERT 0 7.328322 5_Centimeters_Per_Second_(2007)_[720p,BluRay,x264]_-_THORA
INSERT 27 4.841866 Cleanwest 720p
INSERT 21 7.748631 My Bloody Valentine 3D    1080p
INSERT 22 0.779715 Casablana.1942.1080p.BluRay.x264.Codres
INSERT 44 0.534981 Doomsday 720p
INSERT 19 4.277600 Butterfly on a Wheel.2007   720p
REMOVE_ID 19
REMOVE_ID 44
REMOVE_ID 24
PRINT_IN_ORDER actual/34-test.out
INSERT 3 4.017142 Orphanage 720p
INSERT 36 1.232490 Romeo And Juliet   720p
INSERT 37 3.523448 Superman Returns   720p
INSERT 48 6.871528 007.1983.Never.Say.Never.Again.1080p.BluRay.x264-CiNEFiLE
INSERT 9 4.601323 Doomsday 720p
INSERT 54 1.362579 Kingdom of Heaven [2005] [720p]
INSERT 13 6.277796 Next 720P
REMOVE_ID 38
REMOVE_ID 37
REMOVE_ID 22
PRINT_IN_ORDER actual/34-test.out
INSERT 61 1.551531 Body of Lies   720p
INSERT 30 2.641337 007 Thunderball [1965]  [BluRay] [720p] [DTS]
INSERT 8 1.421148 Alien vs Predator 1-2  720p
INSERT 62 8.009853 Epic Movie UNRATED (2007) 720p
INSERT 51 8.120409 Robocop Collection   720p
INSERT 47 3.064770 Shutter   720p
INSERT 32 8.536998 Star Trek [2009] [1080p]
INSERT 7 1.603475 Hitman    720p
REMOVE_ID 30
REMOVE_ID 13
REMOVE_ID 39
REMOVE_ID 10
PRINT_IN_ORDER actual/34-test.out
INSERT 45 5.864247 Pinocchio [1940] [720p]
INSERT 59 5.029338 The Italian Job [2003] [720p]
INSERT 67 7.583676 Lord Of The Ring 2 - The Two Towers (ExtEd) [1080p]
INSERT 57 8.784946 Alone in the dark   720p
INSERT 55 7.274466 Casablana.1942.1080p.BluRay.x264.Codres
INSERT 46 6.561906 American Gangster    720p
INSERT 20 3.448461 Four Christmases 720P
INSERT 18 4.084361 300   720p
INSERT 70 4.019486 Pearl Harbor
REMOVE_ID 47
REMOVE_ID 61
REMOVE_ID 3
REMOVE_ID 42
PRINT_IN_ORDER actual/34-test.out
INSERT 14 8.524305 Watchmen [720p]
INSERT 58 4.775251 Superhero Movie   1080p
INSERT 29 0.426590 Constantine 720p
INSERT 53 7.772330 Robots.WS.AC3.x264.BluRay-PTM   720p
INSERT 41 7.026325 James Bond 20 - Die Another Day [2002] [720p]
INSERT 64 9.492419 Chicken Run [2000] [720p]
INSERT 74 5.730652 Eragon [2006] [720p]
INSERT 71 8.488965 X-Men 1 - 1080p
INSERT 65 8.549737 Cellular.2004.HDTV.720p.AC3.x264-BTT
INSERT 83 9.249135 Ongbak 2003 720p
REMOVE_ID 46
REMOVE_ID 21
REMOVE_ID 11
REMOVE_ID 7
REMOVE_ID 27
PRINT_IN_ORDER actual/34-test.out
INSERT 25 1.120433 Dragonball Evolution [2009] [720p]
INSERT 60 3.126730 Back to the Future Part I [1985] [BluRay] [720p] [AC3] [EN]
INSERT 69 5.796776 Gone In 60 Seconds    1080p
INSERT 92 1.888755 Dolphin    720p
INSERT 73 0.319356 The Orphan [720]
INSERT 16 7.878170 A.Beautiful.Mind.2001.720p.HDTV.x264-S26
INSERT 95 3.814651 Rocky Balboa    720p
INSERT 43 0.411618 Harold & Kumar   720p
INSERT 68 9.725258 PBS
INSERT 84 1.054756 Lord Of The Ring 3 - The Return of the King (ExtEd) [1080p]
INSERT 63 9.937491 Indiana Jones Trilogy   720p
REMOVE_ID 71
REMOVE_ID 16
REMOVE_ID 9
REMOVE_ID 18
REMOVE_ID 65
PRINT_IN_ORDER actual/34-test.out
