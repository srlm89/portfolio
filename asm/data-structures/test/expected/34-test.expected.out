[ 34 0.063821 Beauty And The Beast    720p ]
[ 17 7.955046 Breach 1080p ][ 34 0.063821 Beauty And The Beast    720p ]
[ 2 9.286223 Apollo 13   1080p ][ 10 4.598667 Stardust   720p ][ 17 7.955046 Breach 1080p ][ 34 0.063821 Beauty And The Beast    720p ]
[ 2 9.286223 Apollo 13   1080p ][ 6 1.363797 Click   720p ][ 10 4.598667 Stardust   720p ][ 17 7.955046 Breach 1080p ][ 24 6.745695 Eragon 720p ][ 38 7.518813 SAW V ]
[ 1 9.906953 X-Men 3   1080p ][ 2 9.286223 Apollo 13   1080p ][ 6 1.363797 Click   720p ][ 10 4.598667 Stardust   720p ][ 11 9.508342 Horseman    720p ][ 24 6.745695 Eragon 720p ][ 38 7.518813 SAW V ][ 39 0.406099 Rocky Balboa    720p ][ 42 2.095395 Species   720p ]
[ 0 7.328322 5_Centimeters_Per_Second_(2007)_[720p,BluRay,x264]_-_THORA ][ 1 9.906953 X-Men 3   1080p ][ 2 9.286223 Apollo 13   1080p ][ 6 1.363797 Click   720p ][ 10 4.598667 Stardust   720p ][ 11 9.508342 Horseman    720p ][ 21 7.748631 My Bloody Valentine 3D    1080p ][ 22 0.779715 Casablana.1942.1080p.BluRay.x264.Codres ][ 27 4.841866 Cleanwest 720p ][ 38 7.518813 SAW V ][ 39 0.406099 Rocky Balboa    720p ][ 42 2.095395 Species   720p ]
[ 0 7.328322 5_Centimeters_Per_Second_(2007)_[720p,BluRay,x264]_-_THORA ][ 1 9.906953 X-Men 3   1080p ][ 2 9.286223 Apollo 13   1080p ][ 3 4.017142 Orphanage 720p ][ 6 1.363797 Click   720p ][ 9 4.601323 Doomsday 720p ][ 10 4.598667 Stardust   720p ][ 11 9.508342 Horseman    720p ][ 13 6.277796 Next 720P ][ 21 7.748631 My Bloody Valentine 3D    1080p ][ 27 4.841866 Cleanwest 720p ][ 36 1.232490 Romeo And Juliet   720p ][ 39 0.406099 Rocky Balboa    720p ][ 42 2.095395 Species   720p ][ 48 6.871528 007.1983.Never.Say.Never.Again.1080p.BluRay.x264-CiNEFiLE ][ 54 1.362579 Kingdom of Heaven [2005] [720p] ]
[ 0 7.328322 5_Centimeters_Per_Second_(2007)_[720p,BluRay,x264]_-_THORA ][ 1 9.906953 X-Men 3   1080p ][ 2 9.286223 Apollo 13   1080p ][ 3 4.017142 Orphanage 720p ][ 6 1.363797 Click   720p ][ 7 1.603475 Hitman    720p ][ 8 1.421148 Alien vs Predator 1-2  720p ][ 9 4.601323 Doomsday 720p ][ 11 9.508342 Horseman    720p ][ 21 7.748631 My Bloody Valentine 3D    1080p ][ 27 4.841866 Cleanwest 720p ][ 32 8.536998 Star Trek [2009] [1080p] ][ 36 1.232490 Romeo And Juliet   720p ][ 42 2.095395 Species   720p ][ 47 3.064770 Shutter   720p ][ 48 6.871528 007.1983.Never.Say.Never.Again.1080p.BluRay.x264-CiNEFiLE ][ 51 8.120409 Robocop Collection   720p ][ 54 1.362579 Kingdom of Heaven [2005] [720p] ][ 61 1.551531 Body of Lies   720p ][ 62 8.009853 Epic Movie UNRATED (2007) 720p ]
[ 0 7.328322 5_Centimeters_Per_Second_(2007)_[720p,BluRay,x264]_-_THORA ][ 1 9.906953 X-Men 3   1080p ][ 2 9.286223 Apollo 13   1080p ][ 6 1.363797 Click   720p ][ 7 1.603475 Hitman    720p ][ 8 1.421148 Alien vs Predator 1-2  720p ][ 9 4.601323 Doomsday 720p ][ 11 9.508342 Horseman    720p ][ 18 4.084361 300   720p ][ 20 3.448461 Four Christmases 720P ][ 21 7.748631 My Bloody Valentine 3D    1080p ][ 27 4.841866 Cleanwest 720p ][ 32 8.536998 Star Trek [2009] [1080p] ][ 36 1.232490 Romeo And Juliet   720p ][ 45 5.864247 Pinocchio [1940] [720p] ][ 46 6.561906 American Gangster    720p ][ 48 6.871528 007.1983.Never.Say.Never.Again.1080p.BluRay.x264-CiNEFiLE ][ 51 8.120409 Robocop Collection   720p ][ 54 1.362579 Kingdom of Heaven [2005] [720p] ][ 55 7.274466 Casablana.1942.1080p.BluRay.x264.Codres ][ 57 8.784946 Alone in the dark   720p ][ 59 5.029338 The Italian Job [2003] [720p] ][ 62 8.009853 Epic Movie UNRATED (2007) 720p ][ 67 7.583676 Lord Of The Ring 2 - The Two Towers (ExtEd) [1080p] ][ 70 4.019486 Pearl Harbor ]
[ 0 7.328322 5_Centimeters_Per_Second_(2007)_[720p,BluRay,x264]_-_THORA ][ 1 9.906953 X-Men 3   1080p ][ 2 9.286223 Apollo 13   1080p ][ 6 1.363797 Click   720p ][ 8 1.421148 Alien vs Predator 1-2  720p ][ 9 4.601323 Doomsday 720p ][ 14 8.524305 Watchmen [720p] ][ 18 4.084361 300   720p ][ 20 3.448461 Four Christmases 720P ][ 29 0.426590 Constantine 720p ][ 32 8.536998 Star Trek [2009] [1080p] ][ 36 1.232490 Romeo And Juliet   720p ][ 41 7.026325 James Bond 20 - Die Another Day [2002] [720p] ][ 45 5.864247 Pinocchio [1940] [720p] ][ 48 6.871528 007.1983.Never.Say.Never.Again.1080p.BluRay.x264-CiNEFiLE ][ 51 8.120409 Robocop Collection   720p ][ 53 7.772330 Robots.WS.AC3.x264.BluRay-PTM   720p ][ 54 1.362579 Kingdom of Heaven [2005] [720p] ][ 55 7.274466 Casablana.1942.1080p.BluRay.x264.Codres ][ 57 8.784946 Alone in the dark   720p ][ 58 4.775251 Superhero Movie   1080p ][ 59 5.029338 The Italian Job [2003] [720p] ][ 62 8.009853 Epic Movie UNRATED (2007) 720p ][ 64 9.492419 Chicken Run [2000] [720p] ][ 65 8.549737 Cellular.2004.HDTV.720p.AC3.x264-BTT ][ 67 7.583676 Lord Of The Ring 2 - The Two Towers (ExtEd) [1080p] ][ 70 4.019486 Pearl Harbor ][ 71 8.488965 X-Men 1 - 1080p ][ 74 5.730652 Eragon [2006] [720p] ][ 83 9.249135 Ongbak 2003 720p ]
[ 0 7.328322 5_Centimeters_Per_Second_(2007)_[720p,BluRay,x264]_-_THORA ][ 1 9.906953 X-Men 3   1080p ][ 2 9.286223 Apollo 13   1080p ][ 6 1.363797 Click   720p ][ 8 1.421148 Alien vs Predator 1-2  720p ][ 14 8.524305 Watchmen [720p] ][ 20 3.448461 Four Christmases 720P ][ 25 1.120433 Dragonball Evolution [2009] [720p] ][ 29 0.426590 Constantine 720p ][ 32 8.536998 Star Trek [2009] [1080p] ][ 36 1.232490 Romeo And Juliet   720p ][ 41 7.026325 James Bond 20 - Die Another Day [2002] [720p] ][ 43 0.411618 Harold & Kumar   720p ][ 45 5.864247 Pinocchio [1940] [720p] ][ 48 6.871528 007.1983.Never.Say.Never.Again.1080p.BluRay.x264-CiNEFiLE ][ 51 8.120409 Robocop Collection   720p ][ 53 7.772330 Robots.WS.AC3.x264.BluRay-PTM   720p ][ 54 1.362579 Kingdom of Heaven [2005] [720p] ][ 55 7.274466 Casablana.1942.1080p.BluRay.x264.Codres ][ 57 8.784946 Alone in the dark   720p ][ 58 4.775251 Superhero Movie   1080p ][ 59 5.029338 The Italian Job [2003] [720p] ][ 60 3.126730 Back to the Future Part I [1985] [BluRay] [720p] [AC3] [EN] ][ 62 8.009853 Epic Movie UNRATED (2007) 720p ][ 63 9.937491 Indiana Jones Trilogy   720p ][ 64 9.492419 Chicken Run [2000] [720p] ][ 67 7.583676 Lord Of The Ring 2 - The Two Towers (ExtEd) [1080p] ][ 68 9.725258 PBS ][ 69 5.796776 Gone In 60 Seconds    1080p ][ 70 4.019486 Pearl Harbor ][ 73 0.319356 The Orphan [720] ][ 74 5.730652 Eragon [2006] [720p] ][ 83 9.249135 Ongbak 2003 720p ][ 84 1.054756 Lord Of The Ring 3 - The Return of the King (ExtEd) [1080p] ][ 92 1.888755 Dolphin    720p ][ 95 3.814651 Rocky Balboa    720p ]
